### PR TITLE
fix(FEC-12084): OVP Provider doesn't play entries of partner 0

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

enable partner 0 

Solves FEC-12084

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [x] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/k-provider/ovp/play-source-url-builder.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default class PlaySourceUrlBuilder {
     const {partnerId, entryId, ks, uiConfId, format, protocol, extension, flavorIds} = urlParams;
 
     //verify mandatory params
-    if (!serviceUrlOrigin || !partnerId || !entryId || !format || !protocol) {
+    if (!serviceUrlOrigin || !(partnerId || partnerId === 0) || !entryId || !format || !protocol) {
```
Comment (SivanA-Kaltura) [inline src/k-provider/ovp/play-source-url-builder.js:29]:
```suggestion
    if (!serviceUrlOrigin || !(partnerId > -1) || !entryId || !format || !protocol) {
```

would be shorter

Comment (yairans) [reply src/k-provider/ovp/play-source-url-builder.js:29]: I don't want to break the partnerId as string option

Comment (SivanA-Kaltura) [reply src/k-provider/ovp/play-source-url-builder.js:29]: it can be both numeric and string ?

Comment (yairans) [reply src/k-provider/ovp/play-source-url-builder.js:29]: per the docs it should be a number but string also works so i dont want to break it if an app use it

**Review by SivanA-Kaltura — DISMISSED** [review]

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**